lib/LANraragi/Plugin/Login/EHentai.pm

```diff
@@ -34,22 +35,22 @@ sub do_login {
 
     # Login plugins only receive the parameters entered by the user.
     shift;
-    my ( $ipb_member_id, $ipb_pass_hash, $star ) = @_;
-    return get_user_agent( $ipb_member_id, $ipb_pass_hash, $star );
+    my ( $ipb_member_id, $ipb_pass_hash, $igneous, $star ) = @_;
```

Comment:
Likewise, `$igneous` should be after `$star`.

Reply:
fixed it